rootfs/etc/nginx/lua/metric_emitter.lua

```diff
@@ -0,0 +1,45 @@
+local socket = require('socket')
```

Comment:
why aren't you using already included `ngx.socket`?

Reply:
@ElvinEfendi Didn't realize this was already included 😮